src/components/DiseaseTable.tsx

```diff
@@ -103,11 +107,34 @@ const DiseaseTable = ({
         ),
     };
 
+    const onCellInteraction = (
+        record: UnpackedDiseaseCellLine,
+        index: number | undefined
+    ) => {
+        if (index === undefined) {
+            return {};
+        }
+        return {
+            className: index === hoveredRowIndex ? hoveredRow : "",
+            onMouseEnter: () => setHoveredRowIndex(index),
+            onMouseLeave: () => setHoveredRowIndex(-1),
+            onClick: () => {
+                if (record.status === CellLineStatus.DataComplete)
+                    window.location.href = `/disease-cell-line/AICS-${record.cell_line_id}`;
+            },
+        };
+    };
+
     return (
         <>
             <Table
                 key={diseaseName}
                 className={[container, inProgress ? comingSoon : ""].join(" ")}
+                rowClassName={(record) =>
+                    record.status === CellLineStatus.DataComplete
+                        ? dataComplete
+                        : ""
+                }
                 title={() => (
                     <Flex align="center">
                         <h3 className={tableTitle}>{diseaseName}</h3>
```

Comment:
```suggestion
                        {record.status === CellLineStatus.DataComplete? <Link to={record.cell_line_id}><h3 className={tableTitle}>{diseaseName}</h3></Link> : <h3 className={tableTitle}>{diseaseName}</h3>
```

Don't commit this directly because I'm just writing it in the GitHub UI. also you could define it up above, ie `const cellLineName = ` to keep the JSX from being too messy. But I think it would be good to have true link in here even though it's redundant with the click handler

Reply:
I'm following your thought but I'm pretty sure you mean this to be down on 156-160 where the name of the disease cell line is printed, no where the disease name is, so I'm making the change there.

Reply:
yeah, you're right